tests/avalon_test_framework/data/work_order/work_order_submit.yaml

```diff
@@ -153,7 +153,7 @@ test_workordersubmit_indata_oneValidOtherEmptDataDatahash:
     index: 2
     iv: ''
   - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
-    dataHash: ''
+    dataHash: 'Bangalore'
```

Comment:
the test case is actually test_workordersubmit_indata_oneValidOtherEmptDataDatahash but values for dataHash in both indexes are the same, hence the test was failing.